src/data/plugins.json

```diff
@@ -1227,6 +1227,13 @@
           "link": "https://github.com/SMenigat/cypress-mailhog",
           "keywords": ["email", "mailhog", "test", "commands"],
           "badge": "community"
+        },
+        {
+          "name": "cypress-guerrillamail",
+          "description": "Commands to get random email address, using guerrillamail.com and read its mail box that could be used for automated testing.",
```

Comment:
```suggestion
          "description": "Create and use a randomly-generated email address from Guerrilla Mail.",
```

Reply:
@mike-plummer  Thanks for your suggestions. Do I have to update in my repo first? Do I have to resync? I'm new to contributions.